tex/rules.tex

```diff
@@ -192,6 +192,10 @@ \subsection{Pre-match meeting \label{ref-003}}
 sides. The team not kicking off in the first half of the game will kick off to
 begin the second half of the game.
 
+\added[id=TC]{During the pre-match meeting the referee or their assistant may
```

Comment:
@gngrbrdmn I am not really sure about the wording here -- feel free to let me know if you can think of something better.

Reply:
@RoboCupJuniorTC/soccer-tc  I'm merging this in for the Draft version -- please feel free to add suggest new wording in the draft.